HousingResidentInformationAPI/build.cmd

```diff
@@ -0,0 +1,2 @@
+dotnet restore
+dotnet lambda package --configuration release --framework netcoreapp3.1 --output-package bin/release/netcoreapp3.1/Housing-resident-information-api.zip

```

Comment:
housing-resident-information-api needs to be with lower case "h", as this is what the serverless.yml file is looking for